{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/model/entities.py

```diff
@@ -13,7 +13,7 @@
 log = logging.getLogger(__name__)
 
 # Default cascade setting for parent/child relationships.  Should get set on parent side.
-# More info: https://level12.atlassian.net/wiki/spaces/devs/blog/2015/06/17/14286861/Proper+Configuration+of+SQLAlchemy+Parent+Child+Relationships  # noqa
+# Docs: http://l12.io/sa-parent-child-relationship-config
```

Comment:
I didn't know about `l12.io`.  Maybe it should require HTTPS.

Reply:
My mistake, I meant to have the `https` version, just copied the wrong one.